js/apps/admin-ui/src/user/UserForm.tsx

```diff
@@ -82,7 +82,9 @@ export const UserForm = ({
   const [open, setOpen] = useState(false);
   const [locked, setLocked] = useState(isLocked);
 
-  setValue("requiredActions", user?.requiredActions || []);
+  useEffect(() => {
```

Comment:
Why is this effect needed? Isn't the `user` already backing the form?

Reply:
@jonkoops, I saw this one is still open. Do you need any additional information/changes or can I resolve the feedback?

Reply:
Nope, just need to re-review the code here. Have some high prio things to take care of first.

Reply:
Alright, I'll resolve this then. If you have any new feedback, I'll hear it :)